src/EditorFeatures/VisualBasicTest/ImplementInterface/ImplementInterfaceTests.vb

```diff
@@ -864,22 +864,12 @@ End Class")
         <WorkItem(541092, "http://vstfdevdiv:8080/DevDiv2/DevDiv/_workitems/edit/541092")>
         <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsImplementInterface)>
         Public Async Function TestShowForNonImplementedPrivateInterfaceMethod() As Task
-            Await TestInRegularAndScriptAsync(
+            Await TestMissingInRegularAndScriptAsync(
 "Interface I1
     Private Sub Goo()
 End Interface
 Class A
     Implements [|I1|]
-End Class",
-"Interface I1
-    Private Sub Goo()
-End Interface
-Class A
-    Implements I1
-
-    Public Sub Goo() Implements I1.Goo
-        Throw New System.NotImplementedException()
-    End Sub
```

Comment:
There should be more tests cases, your summary covers at least 3.

Reply:
> There should be more tests cases, your summary covers at least 3.

VB is outside of scope of this PR. I am only adjusting existing affected tests.

Reply:
Also:
- VB doesn't support implicit interface implementations.
- VB doesn't support declaration of static or sealed members within interfaces.

---
In reply to: [268885901](https://github.com/dotnet/roslyn/pull/34365#discussion_r268885901) [](ancestors = 268885901)

Reply:
Ah yea, doi. I was about to respond after your first comment. Just need to update the name of existing test since it's functionally different now.